1.Exploratory_Data_Analysis/0.nomic_cleanup.ipynb

```diff
@@ -187,25 +187,50 @@
     "nomic_df.head(3)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #49.    # four beta of inudcer1, inducer1 concentration, inhibitor, and inhibitor concentration all as 1 beta term
Line #49.    # one beta of inducer1, inducer1 concentration, inhibitor, and inhibitor concentration all as 1 beta term
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>